internal/operands/data-import-crons/reconcile.go

```diff
@@ -0,0 +1,96 @@
+package dataimportcrons
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/selection"
+	cdiv1beta1 "kubevirt.io/containerized-data-importer/pkg/apis/core/v1beta1"
+	"kubevirt.io/ssp-operator/internal/common"
+	"kubevirt.io/ssp-operator/internal/operands"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Define RBAC rules needed by this operand:
+// +kubebuilder:rbac:groups=cdi.kubevirt.io,resources=dataimportcrons,verbs=get;list;watch;create;update;patch;delete
+
+type dataImportCrons struct{}
+
+var _ operands.Operand = &dataImportCrons{}
+
+func GetOperand() operands.Operand {
+	return &dataImportCrons{}
+}
+
+func (c *dataImportCrons) Name() string {
+	return operandName
+}
+
+const (
+	operandName      = "data-import-crons"
+	operandComponent = common.AppComponentTemplating
+)
+
+func (c *dataImportCrons) AddWatchTypesToScheme(s *runtime.Scheme) error {
+	return cdiv1beta1.AddToScheme(s)
+}
+
+func (c *dataImportCrons) WatchClusterTypes() []client.Object {
+	return nil
+}
+
+func (c *dataImportCrons) WatchTypes() []client.Object {
+	return []client.Object{
+		&cdiv1beta1.DataImportCron{},
+	}
+}
+
+func (c *dataImportCrons) Reconcile(request *common.Request) ([]common.ResourceStatus, error) {
+	return common.CollectResourceStatus(request, reconcileDataImportCronsFuncs(request)...)
+}
+
+func (c *dataImportCrons) Cleanup(request *common.Request) error {
+	var existingDataImportCrons cdiv1beta1.DataImportCronList
+	requirement, err := labels.NewRequirement(common.AppKubernetesManagedByLabel, selection.Equals, []string{common.OperatorName})
```

Comment:
Can you please move this to something like `func getLabelRequirement`? Because I think sonarcloud will complain, that this is too big function.